cf-serverd/server_common.c

```diff
@@ -653,7 +649,7 @@ void CfEncryptGetFile(ServerFileGetState *args)
             {
                 if (SendTransaction(conn_info, out, cipherlen + finlen, CF_DONE) == -1)
                 {
-                    Log(LOG_LEVEL_VERBOSE, "Send failed in GetFile. (send: %s)", GetErrorStr());
+                    Log(LOG_LEVEL_ERR, "Protocol transaction broken off in CfEncryptGetFile. Could not send.");
```

Comment:
Please try to wrap at 78 characters where possible (but avoid breaking strings at non-punctuation points, see HACKING.md). 

Also Log messages higher than VERBOSE should not be cryptic, e.g. "CFEncryptGetFile" or "protocol transaction" doesn't mean anything to end users, and this applies to most of the messages you changed.

Finally be careful when changing the level of the message: spamming syslog with errors have been many times an issue. And most times an error must have been printed down the callpath (e.g. in `TLSSend()`) before reaching this point, so no need to print ERR every time we return from a function.

I would rather if you left message changes out of this pull request, in order to be able to test/revert easily.